disco/disco.go

```diff
@@ -204,7 +204,7 @@ func (d *Disco) logInfoState(e *zk.Event) bool {
 }
 
 func (d *Disco) processZkEvent(e *zk.Event) error {
-	d.stateLog.Log(logkey.ZkEvent, e, "disco event seen")
+	d.stateLog.Log(logkey.ZkEvent, e, fmt.Sprintf("disco event for %s", e.Type.String()))
```

Comment:
can we add logKey.Message as the key for message string?

Reply:
Sure. 
We will have to do this to all other logs, may be when we touch those in the future

Reply:
let's fix the ones that we want to use others we can do when necessary